src/frontend-samples/issues-sample/IssuesWidget.tsx

```diff
@@ -454,19 +484,39 @@ const IssuesWidget: React.FunctionComponent = () => {
     ));
   };
 
+  const filterTypes = ["All", "Civil Design", "Clash", "Closeout", "Communication", "Deficiency", "Design", "Field Data", "Issue", "Observation", "Other", "Punchlist", "Risk", "RFI", "Submittal Issue", "Task", "Transmittal Issue"];
+
   return (
     <>
       <div style={{ height: "300px" }}>
+        {/** Only display header when issue isn't selected */}
+        {!currentIssue &&
+          <div className="issue-list-header">
+            <Subheading style={{ margin: "0", padding: "8px 5px", color: "#fff" }}>{`Issues (${currentIssues.length})`}</Subheading>
+            <div className="issue-list-filters">
+              <div className="filter">
+                <span>State</span>
+                <Select options={["All", "Closed", "Open", "Draft"]} value={stateFilter} onChange={(option) => setStateFilter(option.target.selectedOptions[0].value)} />
```

Comment:
I suggest we create at least one closed issue in the sample iModel to make this picker do something in the showcase.

Reply:
A closed issue has been created! Thanks @Josh-Schifter